src/test/java/org/alliancegenome/curation_api/model/entities/ReferenceTest.java

```diff
@@ -1,13 +1,13 @@
 package org.alliancegenome.curation_api.model.entities;
 
-import org.jetbrains.annotations.NotNull;
-import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
 import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.api.Test;
 
 class ReferenceTest {
```

Comment:
I know this wasn't introduced in this PR but I would think this class should be somewhere else as in not under entities because the class is not an `@Entity`

Reply:
moved to `src/test/java/org/alliancegenome/curation_api/identity/ReferenceIdentity.java`, but feel free to suggest something more suitable